addons/web/static/src/core/datetime/datetime_picker.js

```diff
@@ -425,7 +436,7 @@ export class DateTimePicker extends Component {
             additionalMonth: this.additionalMonth,
             maxDate: this.maxDate,
             minDate: this.minDate,
-            showWeekNumbers: !this.props.range,
+            showWeekNumbers: this.props.showWeekNumbers && !this.props.range,
```

Comment:
I'd think that the props should take priority
```suggestion
            showWeekNumbers: this.props.showWeekNumbers ?? !this.props.range,
```

Reply:
makes sense